schema/methods_directory/lin_tetrahedron.json

```diff
@@ -0,0 +1,28 @@
+{
+    "schemaId": "methods-directory-lin-tetrahedron",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Linear tetrahedron method schema",
+    "allOf": [
+        {
+            "$ref": "../method.json"
+        }
+    ],
+    "properties": {
+        "units": {
+            "type": "array",
+            "items": {
+                "anyOf": [
```

Comment:
should this be `oneOf`?

Reply:
I just checked on https://www.jsonschemavalidator.net/. They both work here. I'll switch it to `oneOf` also due to this SO post: https://stackoverflow.com/questions/15396251/correct-json-schema-for-an-array-of-items-of-different-type